### ISSUE TITLE
Primary Key & Foreign Key not shown

### ISSUE THREAD

**Issue by fahmifan** [KarnerTh/mermerd#20]
Hi, I really love this tool. I would like to know is it possible to output the primary key & foreign key label ?

The command I use:
```bash
mermerd --encloseWithMermaidBackticks \
    -c "postgresql://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/${DB_NAME}?sslmode=${DB_SSL_MODE}" \
    --outputFileName ./doc/erd.md \
    --schema public \
    --useAllTables \
    --showAllConstraints
```

I expect something like this:

```mermaid
erDiagram
    users ||--o{ books: own

users {
    uuid id "pk"
    text email
    text password_hash
    text name
    text role
    text status
}

books {
    uuid id "pk"
    uuid owned_by "fk"
}
```

**Comment by KarnerTh** [comment]
Hi, glad you find the tool useful :)

This feature is on the roadmap in the readme - now that I know that someone actually needs it I will add it sooner.
I will inform you when the features is released.

**Comment by KarnerTh** [comment]
This feature will be available in the next release and is active by default. If you want to omit the attributes you can set the config `--omitAttributeKeys`

The only thing that is not supported (by mermaid itself) - you cannot mark a attribute both PK and FK.
As long as mermaid does not support this, mermerd will use PK as key if it is both a PK and FK

**Comment by KarnerTh** [comment]
This feature is now available in [version 0.4.0](https://github.com/KarnerTh/mermerd/releases/tag/v0.4.0)
Please let me know if it does not work as intended
